src/js/correlation/flows/stubs.ts

```diff
@@ -0,0 +1,1869 @@
+import {ZealotPayload} from "../../../../zealot/dist/types"
+
+export const uidAndCommunityResult: ZealotPayload[] = [
```

Comment:
This is a stub for zealot to use in the tests. I just copied and pasted the zjson responses from the network tab in the app after making a query. I'm not sure if this is best place for stubs to live. Open to ideas. It's nice that since this is just a stub for these tests that it lives right next to it. I just don't like that it will be included in the final build. In fact, I don't like like that all our tests are included in the final build.

As an aside, I've been thinking a lot about folder structure. It would be great to have a more scalable structure with a known place for test support files like this (files that are used for tests that don't have a ".test.ts" extension). Then in a build step we could strip out all the tests in the build.